sidebars.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
       items: [
         'pdfs',
         'document_variables_reference',
+        'yaml_anatomy',
```

Comment:
Minor: IDK if this belongs in the PDFs and DOCXs heading?

Reply:
Good question. Created #55 for this